ofs/local/zipstore.py

```diff
@@ -40,28 +41,28 @@ def __init__(self, zipfile, mode="r", compression=ZIP_STORED, allowZip64=False,
         if mode not in ("r", "w", "a"):
             raise RuntimeError('ZOFS() requires mode "r", "w", or "a" (due to underlying ZipFile class)')
         if mode in ("w", "a") and not quiet:
-            print "IMPORTANT: You MUST .close() this ZOFS instance for it to write the ending records in '%s' mode. Otherwise the resultant zip archive will be unreadable." % mode
+            print("IMPORTANT: You MUST .close() this ZOFS instance for it to write the ending records in '%s' mode. Otherwise the resultant zip archive will be unreadable." % mode)
         self.zipfile = zipfile
         self.mode = mode
         self.compression = compression
         self.allowZip64 = allowZip64
         self.hashing_type = hashing_type
         self.quiet = quiet
         if mode == "r" and not is_zipfile(zipfile):
-            raise BadZipArchive, e
+            raise e
```

Comment:
__e__ is an undefined name in this context...  Perhaps __raise BadZipArchive(zipfile + ' is not a valid zipfile.')__

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./ofs/local/zipstore.py:52:19: F821 undefined name 'e'
            raise e
                  ^
```